manage_externals/get_rtma

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+#
+# 1 ./manage_externals/get_rtma
+# 2 ./devbuild.sh
+# 3 source env/wflow_jet.env (use other wflow files for different platforms)
+#   cd regional_workflow/ush
+#   cp config.sh.RTMA_NA_3km config.sh
+#   vi config.sh (modify EXPT_BASEDIR, STMP, PTMP and other variables accordingly)
+# 4 ./generate_FV3LAM_wflow.sh
+#
```

Comment:
manage_externals is a copy of a 3rd-party, standalone utility, and should not include any App-specific bits.

I like the idea of this helper. I think the documentation should go in the README at the top level on how to get/build the RTMA as an app, though. Nobody every really reads the docs in this level of the repo since it's a generic tool.

If you do want to add a helper script instead of just having RTMA folks use the `--externals` flag, you should add it at the top level so that mange_externals remains generic and so that checkout_externals works as expected without tracking down the top directory.

Reply:
@christinaholtNOAA Thanks. I will move get_rtma to the top level. 
As for the comments, your are right, it might be better to add Setup-RTMA.md but I think that will need more writing (similar as Setup-RRFS.md). Since RTMA users are limited now, I just put a very simple command sequence in the comment so that other RTMA folks know how to start. We may add  Setup-RTMA.md later, is that okay? Or do we want a very simple Setup-RTMA.md right now?

Reply:
I think a Setup-RTMA.md sooner rather than later is helpful. Documentation always seems to slip down the list of priorities.